### PR TITLE
feat(deps): migrate react query v5

### PR DESCRIPTION
**Pull request by ReidyT**

### CODE REVIEW

File: cypress/e2e/item/settings/thumbnail.cy.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ describe('Item Thumbnail', () => {
     });
   });
   describe('Upload Thumbnails', () => {
-    it(`upload item thumbnail`, () => {
+    it.only(`upload item thumbnail`, () => {
```
Comment (pyphilia) [inline cypress/e2e/item/settings/thumbnail.cy.ts:28]: Remove.

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "@emotion/styled": "11.13.0",
     "@graasp/chatbox": "3.2.1",
     "@graasp/map": "1.17.0",
-    "@graasp/query-client": "3.22.3",
+    "@graasp/query-client": "github:graasp/graasp-query-client#889-migrate-to-v5",
```
Comment (pyphilia) [inline package.json:23]: Reminder!

**Review by pyphilia — CHANGES_REQUESTED** [review]

**Review by spaenleh — APPROVED** [review]
Looks good ! Just out of curiosity, we do not need to change the usage of `initialIsLoading` ?

**Comment by ReidyT** [comment]
> Looks good ! Just out of curiosity, we do not need to change the usage of `initialIsLoading` ?

No you're right, I forgot this! Thanks for your comment!

**Review by spaenleh — APPROVED** [review]
Very nice !

**Review by pyphilia — APPROVED** [review]
LGTM 🥾

**Pull request merged**